frontend/src/hooks/useAuth.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { AUTH_COOKIE } from 'consts';
+import { useAuth } from '.';
+
+describe('useAuth hook test', () => {
+  it('should return the token when called', () => {
```

Comment:
Maybe use "when token exists"

And then, add another scenario considering "when token does not exist"?

Reply:
When the token not exist is returned undefined, i can coverage this case too.

Reply:
Thanks @edimossilva, your testing teachings are improving this project even though you don't know `nadica de nada` anything about the frontend.

Reply:
Added!

Reply:
Niceeeeee, happy to see that.